### ISSUE TITLE
Log exception when one of the tests fails

### ISSUE THREAD

**Issue by waldekmastykarz** [OfficeDev/generator-office#75]
When one of the tests in `$ gulp test` breaks at the end you get the following error:

```
/Users/usr/github/generator-office/gulpfile.js:129
  $.log(err.toString());
    ^
TypeError: undefined is not a function
    at Stream.handleError (/Users/usr/github/generator-office/gulpfile.js:129:5)
    at Stream.emit (events.js:129:20)
```

Shouldn't https://github.com/OfficeDev/generator-office/blob/master/gulpfile.js#L129 be referring to the function defined in https://github.com/OfficeDev/generator-office/blob/master/gulpfile.js#L113 (so calling `log` rather than `$.log`)?

**Comment by andrewconnell** [comment]
This is an error, but it shouldn't call the local function `log()` because that writes out a successful message in blue all the time. It just needs to call the `$.util.log()` function instead.

**Comment by andrewconnell** [comment]
Fixed in #85
